api/client/client.go

```diff
@@ -201,7 +202,7 @@ func (srv *server) AuthSign(req, id []byte, provider auth.Provider) ([]byte, err
 // It takes the serialized JSON request to send which is required to
```

Comment:
It would be nice to update the documentation to reflect that you are returning the root CA as well now

Reply:
Absolutely